config/initializers/filter_parameter_logging.rb

```diff
@@ -1,4 +1 @@
-# Be sure to restart your server when you modify this file.
-
-# Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password]
+Rails.application.config.filter_parameters += [:password, :value]
```

Comment:
What's `:value` in this context?

Reply:
it's used for key, values when users enter environment variables. value being the potentially sensitive environment variable itself